x-pack/plugins/security_solution/cypress/integration/cases/privileges.spec.ts

```diff
@@ -52,13 +54,13 @@ const testCase: TestCaseWithoutTimeline = {
 describe('Cases privileges', () => {
   before(() => {
     cleanKibana();
+    login();
     createUsersAndRoles(usersToCreate, rolesToCreate);
     visitSecuritySolution();
   });
 
   after(() => {
     deleteUsersAndRoles(usersToCreate, rolesToCreate);
-    cleanKibana();
```

Comment:
Is `cleanKibana()` called after each spec by default?

Reply:
Now it is called in the `before` hook when really needed.